x-pack/plugins/security_solution/public/flyout/entity_details/host_right/content.tsx

```diff
@@ -44,7 +46,12 @@ export const HostPanelContent = ({
   isPreviewMode,
 }: HostPanelContentProps) => {
   const observedFields = useObservedHostFields(observedHost);
-
+  const getSetupStatus = useCspSetupStatusApi({
```

Comment:
do we need this logic in the flyout? I don't think the flyout is smth which stays open longer, so we can rely on calling status API once when it's opened, and not to check it every 10s. In our pages we have this logic for the good onboarding experience, but in my opinion, for the flyout we can make it simpler